priv/riak_kv.schema

```diff
@@ -1034,7 +1034,7 @@
 %% bucket, and warnings about sizing are specially relevant. 
 {mapping, "ttaaefs_allcheck", "riak_kv.ttaaefs_allcheck", [
   {datatype, integer},
-  {default, 24}
+  {default, 0}
```

Comment:
I think that the documentation refers to 24 still, don’t know which is correct but they should probably be conssitent